runtimes/common/src/migrations.rs

```diff
@@ -34,13 +41,18 @@ where
 	T: frame_system::Config,
 {
 	#[cfg(feature = "try-runtime")]
-	fn pre_upgrade() -> Result<sp_std::vec::Vec<u8>, &'static str> {
+	fn pre_upgrade() -> Result<sp_std::vec::Vec<u8>, TryRuntimeError> {
+		use frame_support::ensure;
+
 		log::info!("RemoveInsecureRandomnessPallet::pre_upgrade() checks 🔎");
-		if frame_support::migration::have_storage_value(PALLET_RUNTIME_NAME, PALLET_STORAGE_NAME, b"") {
-			Ok(sp_std::vec::Vec::default())
-		} else {
-			Err("Storage in pallet_insecure_randomness_collective_flip is already empty before migration.")
-		}
+
+		ensure!(
+			frame_support::migration::have_storage_value(PALLET_RUNTIME_NAME, PALLET_STORAGE_NAME, b""),
+			TryRuntimeError::Other(
```

Comment:
Can't we use also here the `log_and_return_error_message` utility function?

Reply:
updated in: [aad9105](https://github.com/KILTprotocol/kilt-node/pull/541/commits/aad9105841f0aaa122c447a7832bc022739cff10)